config/config.yaml

```diff
@@ -1,6 +1,6 @@
 project_code: sage-bionetworks
 profile: {{ var.profile | default("default") }}
 region: {{ var.region | default("us-east-1") }}
-template_bucket_name: bootstrap-awss3cloudformationbucket-zov6lb1ge7ll
+template_bucket_name: {{ environment_variable.TEMPLATE_BUCKET_NAME | default("bootstrap-awss3cloudformationbucket-1apwoyafo211z") }}
```

Comment:
I think cascading configs.yaml is a better approach, https://sceptre.cloudreach.com/latest/docs/stack_group_config.html#cascading-config

basically you'll want to remove `template_bucket_name` from this file and move it to a `develop/config.yaml` file.  here's an example https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/scipool/config/develop/config.yaml